example.cfg

```diff
@@ -86,6 +86,9 @@ COMMIT_MESSAGE_FILTERS = [
 	agito.append_branch_info,  # Append SVN branch/rev info.
 ]
 
+# Configuration for agito.append_branch_info
```

Comment:
Please provide more detail about what this option does, since this example config file serves as a kind of defacto documentation.